core/platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/model/impl/ServerConfImpl.java

```diff
@@ -21,17 +21,20 @@
     private final String ref;
     private final String port;
     private final String protocol;
+    private final String path;
 
-    public ServerConfImpl(String ref, String port, String protocol) {
+    public ServerConfImpl(String ref, String port, String protocol, String path) {
```

Comment:
I think it is still too small to add builder. At least I haven't found a place where I need it.